Remove unsafe addEvent

andersonfaaria opened the pull request:
- Removed AddEvent from two functions of store
- Corrected cults of tibia wall id

dudantas commented:
@liontts Thanks for the contribution.

andersonfaaria committed:
Remove unsafe addEvent

centralized changeParcel logic plus removed unsafe use of addEvent.

guispiller reviewed (changes requested):
When buying Kegs or Decoration Kits, it's not "transforming" the parcel nor the wrapped box into the needed item. It's being just an empty parcel, and a closed box.

andersonfaaria committed:
correcting id of cults of tibia

andersonfaaria committed:
remove addEvent

Concerns: I don't know why this store uses addEvent for everything.
Unfortunately I couldn't fix the addEvent because when I sent the uniqueId to the addEvent, later I cannot retrieve the item through Item(uid) which is VERY STRANGE.

So I'm removing the addEvent but be aware that this might introduce a situation where you can buy the item with packet editor and then closes the store and log out before the coins are removed from your account. (Potentially) or even get the item created before the store checks you're not with it open.
